Classes/New Issue/NewIssueTableViewController.swift

```diff
@@ -189,5 +195,11 @@ final class NewIssueTableViewController: UITableViewController, UITextFieldDeleg
         bodyField.becomeFirstResponder()
         return false
     }
+    
+    /// Called when editing changed on the title field, enable/disable submit button based on title text
+    @objc
+    func textFieldEditingChanged(_ textField: UITextField) {
+        navigationItem.rightBarButtonItem?.isEnabled = textField.hasText
```

Comment:
How does `textField.hasText` handle blank text?

Reply:
Intentionally used `hasText` to match with Github behaviour. It will not handle the blank text. As like you mentioned in #518 

>Blank strings count as a non-empty string despite the API giving an error if you tap the Submit button.

On Submit we will show an error as usual which checks for an empty string

Reply:
While I definitely agree with your point regarding GitHub behaviour, I feel like if we're adding this protection for no text we may as well add it for trimmed text as well - even if we already
 have the validation a little later 😅

Reply:
^ That would make for a better user experience.

Reply:
Just checked, we already have a property `titleText` which checks for trimmed/empty check. So reused that instead of `hasText`.
Will this be fine? @Sherlouk

Reply:
Yup! I knew that would be helpful when I built it 😉